apps/nextjs-website/src/components/molecules/Chat/Chat.tsx

```diff
@@ -0,0 +1,141 @@
+import ChatMessage from '@/components/atoms/ChatMessage/ChatMessage';
+import {
+  Box,
+  IconButton,
+  ListItemIcon,
+  ListItemText,
+  Menu,
+  MenuItem,
+  Stack,
+  useTheme,
+} from '@mui/material';
+import ChatInputText from '../../atoms/ChatInputText/ChatInputText';
+import MenuIcon from '@mui/icons-material/Menu';
+import React, { useEffect, useRef, useState } from 'react';
+import { Delete, History } from '@mui/icons-material';
+
+type ChatProps = {
+  chatMessages: { message: string; sender?: string; timestamp: string }[];
+};
+
+const Chat = ({ chatMessages }: ChatProps) => {
+  const { palette } = useTheme();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const scrollRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    if (scrollRef.current) {
+      scrollRef.current.scrollIntoView({ behavior: 'smooth' });
+    }
+  }, [chatMessages]);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
`handleClick` name isn't very expressive maybe something like `handleChatMenuClick` could be more clear ?